doc/man3/SSL_CTX_set_early_cb.pod

```diff
@@ -0,0 +1,94 @@
+=pod
+
+=head1 NAME
+
+SSL_CTX_set_early_cb, SSL_EARLY_isv2, SSL_EARLY_get0_version, SSL_EARLY_get0_random, SSL_EARLY_get0_session, SSL_EARLY_get0_ciphers, SSL_EARLY_get0_compression_methods, SSL_EARLY_get0_ext - callback functions for early server-side ClientHello processing
+
+=head1 SYNOPSIS
+
+ typedef int (*ssl_early_cb_fn) (SSL *s, SSL_EARLY *ectx,
+                                 const unsigned char *clienthello, size_t ch_len,
+                                 int *al, void *arg);
+ void SSL_CTX_set_early_cb(SSL_CTX *c, ssl_early_cb_fn cb, void *arg);
+ int SSL_EARLY_isv2(SSL *s, SSL_EARLY *ectx);
```

Comment:
If the function name is `SSL_EARLY_*` then the `SSL_EARLY` argument should be the first IMO